src/codeu/chat/server/Server.java

```diff
@@ -64,6 +65,8 @@
   private final Relay relay;
   private Uuid lastSeen = Uuid.NULL;
 
+  private static final ServerInfo info = new ServerInfo();
```

Comment:
static final constants should be named in all upper case (so 'INFO' instead of 'info')